packages/browser/src/index.test.ts

```diff
@@ -97,4 +102,36 @@ describe('LogtoClient', () => {
       expect(window.location.toString()).toEqual(mockedSignInUri);
     });
   });
+
+  describe('signOut', () => {
+    beforeEach(() => {
+      localStorage.setItem(idTokenStorageKey, 'id_token_value');
+      localStorage.setItem(refreshTokenStorageKey, 'refresh_token_value');
+    });
+
+    test('should call token revocation endpoint with requester', async () => {
+      const logtoClient = new LogtoClient({ endpoint, clientId, requester });
+      await logtoClient.signOut(postSignOutRedirectUri);
```

Comment:
will this send a real request?

Reply:
also should cover request failed?

Reply:
> will this send a real request?

No, the `requester` function is mocked at the very beginning of this test suite.

> also should cover request failed?

I'm afraid not, as we do not want to block sign-out flow even if the revocation is failed, so there is zero handling in the `try-catch` block of the `revoke` request.

Reply:
yes, i mean we should cover "even if the request failed but sign out is still non-blocking" case

Reply:
OK, will add that part shortly